src/main/groovy/wooga/gradle/unity/tasks/RunCSScript.groovy

```diff
@@ -0,0 +1,82 @@
+package wooga.gradle.unity.tasks
+
+import org.gradle.api.file.RegularFile
+import org.gradle.api.file.RegularFileProperty
+import org.gradle.api.provider.Property
+import org.gradle.api.provider.Provider
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.InputFile
+import wooga.gradle.unity.UnityTask
+
+import java.nio.file.Files
+
+class RunCSScript extends UnityTask {

```

Comment:
I would be more verbose with the name here. `RunCsharpScript`? Or follow the same naming convention for tasks in our wdks. I think would be `ExecuteCsharpScript`?